_im-plugin/ism/policies.md

```diff
@@ -438,6 +439,133 @@ Parameter | Description | Type | Required
+                                            "timezone": "America/Los_Angeles"
+                                        }
+                                    },
+                                    {
+                                        "terms": {
+                                            "source_field": "customer_gender",
+                                            "target_field": "customer_gender"
+                                        }
+                                    },
+                                    {
+                                        "terms": {
+                                            "source_field": "day_of_week",
+                                            "target_field": "day_of_week"
+                                        }
+                                    }
+                                ],
+                                "metrics": [
+                                    {
+                                        "source_field": "taxless_total_price",
+                                        "metrics": [
+                                            {
+                                                "sum": {}
+                                            }
+                                        ]
+                                    },
+                                    {
+                                        "source_field": "total_quantity",
+                                        "metrics": [
+                                            {
+                                                "avg": {}
+                                            },
+                                            {
+                                                "max": {}
+                                            }
+                                        ]
+                                    }
+                                ]
+                            }
+                        }
+                    }
+                ],
+                "transitions": []
+            }
+        ]
+    }
+}
+````
+
+#### Request Fields
+
+The following request fields are required.
+
+Field Parameter | Data Type | Description | Required
+:--- | :--- |:--- |:---
+description | String | The description name of the ISM policy. | No
+default_state | String | The state that the ISM policy runs by default. Set the description to `rollup` for the ISM rollup policy. | Yes
+states | List of objects | Define the states to be used in the ISM rollup policy. | Yes
+states.name | List of objects | The description name of the state within the ISM policy. In this case, you will need to match the `default_state` description of "`rollup`." | Yes
+states.actions | List of objects | Define the actions to be run by the state. | Yes
+states.transition | List of objects | Define the conditions required to move on to the next state. When left blank, the policy completes its management of the index. | Yes
```

Comment:
These fields are general fields for all ISM policy, and we have explained in the above Policies section.